src/components/ActionMenu/ActionMenu.scss

```diff
@@ -2,6 +2,10 @@
 @import './variables';
 
 .ActionMenu {
+  @include print-hidden;
+  // Needed for JS calculations
+  width: 100%;
```

Comment:
For some reason web is rendering this container where Polaris isn't I've compared in the old DL and this change doesn't affect it